plugin/src/main/java/org/screamingsandals/bedwars/special/ArrowBlocker.java

```diff
@@ -68,7 +69,11 @@ public void activate() {
             if (item.getAmount() > 1) {
         		item.setAmount(item.getAmount() - 1);
         	} else {
-        		player.getInventory().remove(item);
+        		if(player.getInventory().getItemInOffHand().equals(item)) {
+        			player.getInventory().setItemInOffHand(new ItemStack(Material.AIR));
+        		}else {
```

Comment:
Also please match our formatting :)

Reply:
Is it the tabs thing? if so I already fixed

Reply:
Nope, it is the if(something). We use if (something) :)

Reply:
Ok I fixed it

Reply:
Now else and catch and formatting is fine 😂

Reply:
oof Forgot about that

Reply:
Done